spec/Section 3 -- Type System.md

```diff
@@ -339,19 +339,9 @@ Scalar types represent primitive leaf values in a GraphQL type system. GraphQL
 responses take the form of a hierarchical tree; the leaves of this tree are
 typically GraphQL Scalar types (but may also be Enum types or {null} values).
 
-GraphQL provides a number of built-in scalars (see below), but type systems can
-add additional scalars with semantic meaning. For example, a GraphQL system
-could define a scalar called `Time` which, while serialized as a string,
-promises to conform to ISO-8601. When querying a field of type `Time`, you can
-then rely on the ability to parse the result with an ISO-8601 parser and use a
-client-specific primitive for time. Another example of a potentially useful
-custom scalar is `Url`, which serializes as a string, but is guaranteed by
-the service to be a valid URL.
-
-```graphql example
-scalar Time
-scalar Url
-```
+GraphQL provides a number of built-in scalars which are fully defined in the 
+sections below, however type systems may also add additional custom scalars to 
+introduce additional semantic meaning (see below).
```

Comment:
I agree with moving custom scalars to their own section, but I find the resulting paragraph here a bit confusing as it refers to two separate things "below" in a way that's somewhat ambiguous.

Reply:
Good point. Let me think about how to clarify